packages/app/src/elements/item-view.ts

```diff
@@ -77,6 +77,9 @@ export class ItemView extends Routing(StateMixin(LitElement)) {
     @state()
     private _fields: Field[] = [];
 
+    @state()
+    private _expiresAt?: Date;
+
```

Comment:
This is in line with the rest of the editing logic. Basically we remember the updated property and don't apply it before the user hits save. This way the user can add/remove the expiration date in edit mode and hit cancel without the change actually being applied.